src/mlpack/methods/det/dtree_impl.hpp

```diff
@@ -95,7 +93,7 @@ void ExtractSplits(std::vector<std::pair<ElemType, size_t>>& splitVec,
                    const size_t minLeafSize)
 {
   // It's common sense, but we also use it in a check later.
-  Log::Assert(minLeafSize > 0);
+  mlpack::Log::Assert(minLeafSize > 0);
```

Comment:
I am not sure, but I think that you can revert most of the changes to this file, since these functions are now in the `mlpack::det` namespace.